resources/js/app.js

```diff
@@ -1,4 +1,5 @@
 import './bootstrap';
+import jsQR from "jsqr";
```

Comment:
This is necessary to use the library correctly. Since it is installed via `npm install jqr`, it means it can be used this way (no need to download the library file)